media/js/app/assetmgr/collectionwidget.js

```diff
@@ -256,9 +256,9 @@ CollectionWidget.prototype.mapEvents = function() {
                 success: function(json, textStatus, xhr) {
                     self.signalInsert(assetId, json.annotation.id);
                     self.$modal.modal('hide');
-                    return;
                 }
             });
+            return;
```

Comment:
This looks fine really, but I was under the assumption that an empty `return` at the end of an event handler is unnecessary. In my mind both of these should be removed - I'm aware of returning `false` from an event handler as discussed here: http://stackoverflow.com/a/4379459/173630 but what does this `return` do?